quickwit/quickwit-indexing/src/actors/merge_planner.rs

```diff
@@ -38,13 +42,24 @@ pub struct MergePlanner {
     partitioned_young_splits: HashMap<u64, Vec<SplitMetadata>>,
     merge_policy: Arc<dyn MergePolicy>,
     merge_split_downloader_mailbox: Mailbox<MergeSplitDownloader>,
+    ongoing_merge_operations_inventory: Inventory<MergeOperation>,
 }
 
 #[async_trait]
 impl Actor for MergePlanner {
-    type ObservableState = ();
-
-    fn observable_state(&self) -> Self::ObservableState {}
+    type ObservableState = MergePlannerState;
+
+    fn observable_state(&self) -> Self::ObservableState {
+        let ongoing_merge_operations = self
+            .ongoing_merge_operations_inventory
+            .list()
+            .iter()
+            .map(|tracked_operation| tracked_operation.as_ref().clone())
```

Comment:
That's neat.

The supervised state of all actors in the universe is accessible. We should put that in the REST API.

Reply:
I prepared it in a next PR :)